src/data/nav/aitransport.ts

```diff
@@ -16,6 +16,15 @@ export default {
           link: '/docs/ai-transport',
           index: true,
         },
+        {
```

Comment:
Just need to move this to its own section so we don't have a dropdown:

It should look like:

```
  content: [
    {
      name: 'Introduction',
      pages: [
        {
          name: 'About AI Transport',
          link: '/docs/ai-transport',
          index: true,
        },
      ],
    },
    {
      name: 'Token streaming',
      pages: [
        {
          name: 'Message per response',
          link: '/docs/ai-transport/features/token-streaming/message-per-response',
        },
      ],
    },
  ],
  ```